clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -2823,3 +2823,84 @@ void tools::addOffloadCompressArgs(const llvm::opt::ArgList &TCArgs,
     CmdArgs.push_back(
         TCArgs.MakeArgString(Twine("-compression-level=") + Arg->getValue()));
 }
+
+void tools::addMCModel(const Driver &D, const llvm::opt::ArgList &Args,
```

Comment:
Is this taken verbatim from Clang? Would you mind documenting that in the summary?

Reply:
Yes, it is a direct copy, aside from not using a "cached" TripleStr and instead calling getTriple() on the Triple. [I just compared the two versions].